### PR TITLE
Add Basic Replugged Support

### PR DESCRIPTION
**Pull request by natowb**
Hey @orblazer,
Have gotten replugged setup for this project.

- it uses the `nordic.theme.css` file as its src as replugged can only have a single css as source per repository
- would be trivial to fork the project to add uniform support then keep it up to date based off this repo.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "id": "dev.natowb.nordic",
+  "name": "Nordic",
+  "description": "An full themed discord with Nord palette (https://www.nordtheme.com/)",
+  "author": [
+   {
+      "name": "orblazer",
+      "discordID": "NEED_DISCORD_ID",
```
Comment (natowb) [inline manifest.json:8]: If you want to link your discord id youll need to add it here

**Comment by orblazer** [comment]
Hello,
Thanks for submitting that PR, i will check about that and check if some point of the theme is broken in few weeks.

**Comment by natowb** [comment]
> Hello,
> Thanks for submitting that PR, i will check about that and check if some point of the theme is broken in few weeks.

No worries, let me know if you would prefer the replugged variant in its own repository. I'm happy to keep fork and manage it.

I'm currently using the release it generated and it looks good to me

**Comment by natowb** [comment]
> Hello, I work on that based on your PR, but with few tweaks. Before i fully include it, could you tell me if the `uniform` variant could be easily included or we need an other repo for that ?
> 
> Thanks.

from my understanding replugged requires the github repo to contain a manifest in the root. so it wouldnt be possible to have more than one in a single repo.

**Pull request closed without merging**